gallery/how_to/work_with_microtvm/micro_custom_ide.py

```diff
@@ -0,0 +1,358 @@
+#
+# #. In the "Board Selector" tab, type in the board name "nucleo-l4r5zi" in the "Commercial Part Number" text box. Select the board from the list of boards that appear on the right side of the screen and click "Next".
+#
+# #. Type in your project name (for example microtvm_vww_demo). We are using the default options. (Target Language: C, Binary Type: Executable, Project Type: STM32Cube). Click "Finish".
+#
+# #. A text box would appear that asks you if you want to "Initialize all the peripherals with their default mode?". click "Yes". This would generate the project and open the device configuration tool where you can use the GUI to setup the peripherals. By default the USB, USART3 and LPUART1 are enabled, as well as a few GPIOs.
+#
+# #. We will use LPUART1 to send data to the host pc. From the connectivity section, select the LPUART1 and set the "Baud Rate" to 115200 and the "Word Length" to 8. Save the changes and click "Yes" to regenerate the initialization code. This should regenerate the code and open your main.c file. You can also find main.c from the Project Explorer panel on the left, under microtvm_vww_demo -> Core -> Src.
+#
+# #. For sanity check, copy the code below and paste it in the "Infinite loop (aka. While (1) ) section of the main function.
+#
+#    * Note: Make sure to write your code inside the sections marked by USER CODE BEGIN <...> and USER CODE END <...>. The code outside these sections get erased if you regenerate the initialization code.
+#
+#        .. code-block:: c
+#
+#         HAL_GPIO_TogglePin(LD2_GPIO_Port, LD2_Pin);
+#         HAL_UART_Transmit(&hlpuart1, "Hello World.\r\n", 14, 100);
+#         HAL_Delay(1000);
+#
+# #. from the menu bar, select Project -> Build (or right click on project name and select Build). This should build the project and generate the .elf file. Select Run -> Run to download the binary on your MCU. If the "Edit Configuration" window opens, just click "OK".
+#
+# #. Open the terminal console on your host machine. On Mac you can simply use the "screen <usb_device> 115200" command. An LED should blink on the board and the string "Hello World" should print out on your terminal console every second.
+#
+
+######################################################################
+# Import the model to the generated project
+# ------------------------------------------
+#
+# To integrate the compiled model into the generated project, follow these steps:
+#
+# #. Extract the tar file and include it in the project
+#
+#    * Open the project Properties. (by right clicking on the project name and selecting "Properties" or by selecting Project -> Properties from the menu bar).
+#    * Select C/C++ General -> Paths and Symbols. Select the Source Location tab.
+#    * If you extract the model inside the project folder on your disk, click "Add Folder" and select the "model" folder. (You might need to right click on the project name and select "Refresh" before it appears.)
```

Comment:
I couldn't get the "Add Folder" method to work on MacOS, even after restarting the IDE.

Reply:
Not sure why it didn't work for you. Please make sure the extracted "model" folder is inside your project folder, and refresh the project explorer.

Reply:
Yea doing the "Refresh" made it show up